src/NzbDrone.Core/Indexers/Newznab/NewznabRequestGenerator.cs

```diff
@@ -404,17 +404,45 @@ public virtual IndexerPageableRequestChain GetSearchRequests(AnimeEpisodeSearchC
                         "search",
                         $"&q={NewsnabifyTitle(queryTitle)}+{searchCriteria.AbsoluteEpisodeNumber:00}"));
 
-                    if (Settings.AnimeStandardFormatSearch && searchCriteria.SeasonNumber > 0 && searchCriteria.EpisodeNumber > 0)
+                    if (Settings.AnimeStandardFormatSearch && searchCriteria.SeasonNumber > 0)
                     {
-                        AddTvIdPageableRequests(pageableRequests,
-                            Settings.AnimeCategories,
-                            searchCriteria,
-                            $"&season={NewznabifySeasonNumber(searchCriteria.SeasonNumber)}&ep={searchCriteria.EpisodeNumber}");
-
-                        pageableRequests.Add(GetPagedRequests(MaxPages,
-                            Settings.AnimeCategories,
-                            "tvsearch",
-                            $"&q={NewsnabifyTitle(queryTitle)}&season={NewznabifySeasonNumber(searchCriteria.SeasonNumber)}&ep={searchCriteria.EpisodeNumber}"));
+                        if (searchCriteria.IsSeasonSearch)
+                        {
+                            if (searchCriteria.SeasonNumber < 10)
```

Comment:
With the `season` query param Sonarr doesn't need to pad the season number, the indexer should handle it correctly regardless and in the case of Prowlarr and Jackett they know best how to query the different indexers. This is the same as `ep`, which isn't padded.

Reply:
We removed on what we think the unnecessary code was since the indexers should already know best.